packages/desktop/components/buttons/NetworkDrawerButton.svelte

```diff
@@ -1,7 +1,17 @@
 <script lang="ts">
-    import { FontWeight, Text, NetworkIcon } from '@ui'
-    import { DrawerId, closeDrawer, drawerState, openDrawer } from '@desktop/auxilary/drawer'
+    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { NetworkHealth, chainStatuses, networkStatus } from '@core/network'
     import { activeProfile } from '@core/profile/stores'
+    import { DrawerId, closeDrawer, drawerState, openDrawer } from '@desktop/auxilary/drawer'
+    import { FontWeight, Icon, NetworkIcon, Text } from '@ui'
+
+    $: isAnyChainDisconnected = Object.values($chainStatuses ?? [])
+        ?.map(({ health }) => health)
+        ?.some((status) => status === NetworkHealth.Disconnected)
```

Comment:
Why not check directly against the `health` property and get rid of the extra map?

```suggestion
        ?.some(({health}) => health === NetworkHealth.Disconnected)
```